src/streams/register_nack.rs

```diff
@@ -275,6 +280,18 @@ mod test {
         assert!(reg.nack_reports().is_none());
     }
 
+    #[test]
+    fn nack_test_huge_seq_gap_no_hang() {
+        let mut reg = NackRegister::new();
+
+        let before = Instant::now();
+
+        reg.update(0.into());
+        reg.update(18446744073709551515.into());
+
+        assert!(before.elapsed().as_millis() < 1000);
```

Comment:
I really dislike these kinds of tests, because if the CI is slow, you get random errors. And no, it doesn't matter how huge we make 1000, it's still a flakey test because it can fail despite being correct. This is why str0m always drives time from outside and have no internal timers.